### PR TITLE
Setup KF Model Registry GH tests

### PR DESCRIPTION
**Pull request by lampajr**
**Which issue is resolved by this Pull Request:**
Resolves #

Followup of https://github.com/kubeflow/manifests/issues/2631

**Description of your changes:**

Add GH test for Kubeflow Model Registry component.

**Checklist:**
- [x] Unit tests pass:
  **Make sure you have installed kustomize == 5.2.1+**
    1. `make generate-changed-only`
    2. `make test`

### CODE REVIEW

**Comment by lampajr** [comment]
Note that at the moment there is no `apps/model-registry/upstream` content.

Do you want to add that from `main` or [`v0.2.0-alpha`](https://github.com/kubeflow/model-registry/releases/tag/v0.2.0-alpha) branch?

**Comment by lampajr** [comment]
fyi @tarilabs @rimolive

**Comment by tarilabs** [comment]
> Do you want to add that from `main` or [`v0.2.0-alpha`](https://github.com/kubeflow/model-registry/releases/tag/v0.2.0-alpha) branch?

I believe it will be meaningful to start tracking https://github.com/kubeflow/model-registry/releases/tag/v0.2.0-alpha
let me know if you need a proper branch or tag is sufficient like atm

**Comment by lampajr** [comment]
> I believe it will be meaningful to start tracking https://github.com/kubeflow/model-registry/releases/tag/v0.2.0-alpha 

+1, if we all agree on that I will add the content as part of this PR
 
> let me know if you need a proper branch or tag is sufficient like atm

The script to sync the manifest requires a generic git _ref_, therefore the tag is absolutely fine.

File: .github/workflows/model_registry_test.yaml
Comment (lampajr) [inline .github/workflows/model_registry_test.yaml:1]: Just for the records, if we change or improve these tests, please reflect the same changes here https://github.com/kubeflow/model-registry/blob/main/.github/workflows/build-image-pr.yml

**Review by tarilabs — COMMENTED** [review]
/lgtm thanks @lampajr

**Comment by rimolive** [comment]
/lgtm

cc @juliusvonkohout

**Comment by rimolive** [comment]
@lampajr Thank you for your contribution! After discussing with Manifests WG leads we decided to cherry-pick the functional tests in #2697 so we add the manifest and run the tests.

Looking for your ack so I can close this PR in favor of #2697.

**Comment by lampajr** [comment]
> @lampajr Thank you for your contribution! After discussing with Manifests WG leads we decided to cherry-pick the functional tests in #2697 so we add the manifest and run the tests.
> 
> Looking for your ack so I can close this PR in favor of #2697.

That's fine for me, thanks @rimolive !

**Comment by lampajr** [comment]
Closing this in favor of https://github.com/kubeflow/manifests/pull/2697

**Pull request closed without merging**